docs/compatibility.md

```diff
@@ -124,7 +124,9 @@ On the other hand, HTTPX uses [HTTPCore](https://github.com/encode/httpcore) as
 
 `requests` omits `params` whose values are `None` (e.g. `requests.get(..., params={"foo": None})`). This is not supported by HTTPX.
 
-## Determining the next redirect request
+## Redirection
```

Comment:
Should we consider a dedicated section? The next info about `response.next` is really separate and the current "Determining the next redirect request" is well-fitted for it.

Eg we could use a new section for this info, named:

```markdown
## HEAD redirection
```

Reply:
@florimondmanca Done. Amended the commit and separated this part as "HEAD redirection".